BREAKING.md

```diff
@@ -103,6 +104,13 @@ The following members of the `@fluidframework/driver-utils` package have been de
 
 -   `waitForConnectedState`
 
+### Aqueduct members deprecated
+
+The following members of the `@fluidframework/aqueduct` package have been deprecated and will be removed in an upcoming release:
+
+-   `waitForAttach()`
+    - Instead inspect the IFluidDataStoreRuntime's attachState property, and await the "attached" event if not attached.
```

Comment:
I'm not sure about that. I think the better guidance -there should be no scenarios where knowing if object is attached or not is required. We are removing and discouraging any direct manipulations in this space due to GC, so attachment happens only as result of using handles, and users of the system should not care how long it takes to propagate local changes to service, with exception of checking dirty state of the document where appropriate. Otherwise, it should not matter if attachment process happened or not.

Reply:
In the Loop usage they use it to ensure they've waited until the point where they can expect a real result back from `getAbsoluteUrl()`.  I think this is probably still a legitimate use under current circumstances, but agree with your statement in general.  I'll update the breaking notice to more strongly discourage use.

Reply:
@agarwal-navin  - FYI (given your other PRs in this space, including deprecation of various methods on handle).
I think users should be able to use uri after attachment process started (not when it finished). That said, feels like getAbsoluteUrl() should be deprecated as well :)

Reply:
For `getAbsoluteUrl()` deprecation - It should happen as part of the request API removal effort. It is one of the blockers right now where apps are asking for an alternative for sharing components with external apps.

Reply:
And yes, checking for !Detached should be sufficient since that means the attachment process has started.